johnsonandjohnson/Bodiless-JS
# feat(filtering): Enable optional hiding of filterable item when no ta…

## Changes
- Add a `showWhenNoTagSelected' prop to the filterable item which controls its visibility when no tag is selercted.

## How to test
On /filter-item page there is now a chackbox to enable/disable this functionality.  With checkbox deselected, you should see no items when no tags are selected.  With checkbox selected, you sould see all.  Behavior when one or more tags is selected should be the same.
